dvc/utils/stage.py

```diff
@@ -1,19 +1,32 @@
+import yaml
 from ruamel.yaml import YAML
 from ruamel.yaml.error import YAMLError
 
+try:
+    from yaml import CSafeLoader as SafeLoader
```

Comment:
is it installed automatically on most systems?

Reply:
PyYAML for Windows has wheels, which include binaries. I didn't test how it works though. For other systems you will `libyaml-dev` installed to use C. Was not installed in my Ubuntu, so one needs to:
```bash
sudo apt install libyaml-dev
```
Before installing PyYAML, otherwise it won't be linked.

Pure python PyYAML still works 2x fast as `ruamel.yaml`. We might also poke `PyYAML` author to create all the wheels.

Reply:
That is quite a requirement, and won't be easy to explain to the users. This will result in dvc stage collection staying slow for pretty much everyone, except a few guys who will discover that they need to install libyam-dev.

Reply:
Poking PyYAML guy and helping him is still an option. Will work on top of this seamlessly.

Reply:
2x sounds good and we can def ping the guys (let's do this right now?)